java/Dockerfile

```diff
@@ -4,16 +4,17 @@
 ARG MAVEN_VERSION=3.8
 ARG JAVA_VERSION=16
 FROM maven:${MAVEN_VERSION}-eclipse-temurin-${JAVA_VERSION} AS java
-
+WORKDIR /cucumber
 
 # Dummy stage for generated code, overriden in main build
 FROM scratch AS schema-codegen
 
 
 FROM java AS with-dependencies
 
-COPY --link pom.xml maven-versions-rules.xml .
-RUN mvn install
+COPY --link pom.xml .
+# verify loads additional dependencies without running the tests (they aren't copied in yet)
+RUN mvn dependency:go-offline verify
```

Comment:
This `verify` command will run test. I don't think it's needed here

Reply:
It triggers some extra dips to load but pas per the comment because we haven't copied the sources no tests run

Reply:
Aan gotcha! 😄